rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -26,14 +43,32 @@
       });
     }
 
+    function pollForResponses() {
+      $.get(responsesUrl).then(function(newState) {
+        state = Object.assign({}, state, newState);
+        render();
+        if (state.is_open) {
+          setTimeout(pollForResponses, POLLING_MILLIS);
+        }
+      }).fail(function () {
+        // TODO: try again?
+        console.error("Error retrieving response data");
+      });
+    }
+
     $(function($) { // onLoad
       var block = $element.find(rapidTopLevelSel);
       var isOpen = block.attr('data-open') === 'True';
       var isStaff = block.attr('data-staff') === 'True';
-      render({
+      Object.assign(state, {
```

Comment:
I meant something like this in my earlier comment:

```javascript
Object.assign(state, {
  is_open: block.attr('data-open') === 'True',	
  is_staff: block.attr('data-staff') === 'True'
});
```

2 variables above would be removed